design-toolkit/components/src/components/button/styles.ts

```diff
@@ -13,7 +13,7 @@
 import { tv } from '@/lib/utils';
 
 export const ButtonStylesDefaults = {
-  color: 'info',
+  color: 'mono-muted',
```

Comment:
We can actually remove this property because we don't implement color as a `tv` variant